### PR TITLE
Added console command #247

### PR DESCRIPTION
**Pull request by grallc**
Added console command (#247) to open the application's console from CLI and removed dupllicated addonName arg

### CODE REVIEW

**Comment by hsablonniere** [comment]
@grallc Hey Corentin, thanks very much for you contribution :tada:

* I have a few tiny remarks but it looks very good :wink:
* FYI: The release of this code may take some time since we're in the middle of a big change on the releasing process...

File: bin/clever.js [hunk 1]
```diff
@@ -501,6 +500,13 @@ function run () {
     options: [opts.alias],
   }, open);
 
+    // CONSOLE COMMAND
+    const console = lazyRequireFunctionWithApi('../src/commands/console.js');
```
Comment (hsablonniere) [inline bin/clever.js]: I'm afraid this naming could confuse people with the already existing global variable `console`.

* Could you rename this?
* Maybe something like `consoleModule` or `consoleFn`

File: src/commands/console.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const AppConfig = require('../models/app_configuration.js');
+const Domain = require('../models/domain.js');
+const handleCommandStream = require('../command-stream-handler.js');
+const Logger = require('../logger.js');
+const OpenBrowser = require('../open-browser.js');
+
+function console (api, params) {
```
Comment (hsablonniere) [inline src/commands/console.js]: Same here :smile:

I'm afraid this naming could confuse people with the already existing global variable `console`.

* Could you rename this?
* Maybe something like `consoleModule` or `consoleFn`

File: src/commands/console.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  const { alias } = params.options;
+  const s_open = AppConfig.getAppData(alias)
+    .flatMapLatest(({ app_id, org_id }) => {
+      Logger.println('Opening the console in your browser');
+      return OpenBrowser.openPage('https://console.clever-cloud.com/users/me/applications/' + app_id);
```
Comment (hsablonniere) [inline src/commands/console.js]: I haven't tested but I think it won't work for apps in organizations.
We miss a test on the `org_id` to build the right URL in any case.

I also think we should use template strings for this:

```js
const appUrl = `https://console.clever-cloud.com/users/me/applications/${app_id}`;
```

**Comment by grallc** [comment]
Just pushed the amended commit.
Sorry for the organisation, I didn't know their existence... It's fixed now!
Thanks you for your tips ! :smile:

File: src/commands/console.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  const { alias } = params.options;
+  const s_open = AppConfig.getAppData(alias)
+    .flatMapLatest(({ app_id, org_id }) => {
+      Logger.println('Opening the console in your browser');
+      let path;
+
+      if (org_id) {
+        path = `organisations/${org_id}`;
+      }
+      return OpenBrowser.openPage(`https://console.clever-cloud.com/${path || 'users/me'}/applications/${app_id}`);
```
Comment (hsablonniere) [inline src/commands/console.js]: What do you think of this?

```js
const path = (org_id != null) ? `organisations/${org_id}` : 'users/me';
return OpenBrowser.openPage(`https://console.clever-cloud.com/${path}/applications/${app_id}`);
```

Comment (grallc) [reply src/commands/console.js]: Oh thanks!
I didn't know this syntax :sweat_smile:

File: package.json [hunk 5]
```diff
@@ -13,7 +13,7 @@
   "author": "Rodolphe Belouin <rodolphe.belouin@gmail.com>",
   "license": "MIT",
   "bin": {
-    "clever": "./bin/clever.js",
+    "clever-test": "./bin/clever.js",
```
Comment (hsablonniere) [inline package.json]: Ooops

Comment (grallc) [reply package.json]: One more...
Really sorry for the spam :(

Comment (hsablonniere) [reply package.json]: This is not spam. This is iterative contribution :wink: which is always welcome.

**Comment by hsablonniere** [comment]
@grallc I'll test this in the afternoon and merge it if everything is OK.

**Comment by hsablonniere** [comment]
@grallc I just merged your commit manually (so I could remove the diff on the `package-lock.json`. Thanks again for you contribution!!!

:tada:

**Pull request closed without merging**